### PR TITLE
Upgrade sabberworm/php-css-parser to latest v8.3.0

### PR DESCRIPTION
**Pull request by fitztrev**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     }
   },
   "require": {
-    "sabberworm/php-css-parser": "8.1.*"
+    "sabberworm/php-css-parser": "8.3.*"
```
Comment (PowerKiKi) [inline composer.json]: This should allow allow non-breaking updates in the future too, like so:

```suggestion
    "sabberworm/php-css-parser": "^8.3"
```

**Comment by fitztrev** [comment]
Changed with your suggestion. Thanks

**Comment by m-bosch** [comment]
Can this be merged. We cant upgrade to Laravel 6 because some dependencies have deep relations to this packages and require the 8.3 version. thank you

**Comment by PowerKiKi** [comment]
@PhenX according to your GitHub activity it seems you might have moved to C#. If you don't have the time, or interest, to maintain this lib anymore, could you please consider adding another contributor or letting us know that we should fork ?

**Comment by m-bosch** [comment]
@PowerKiKi do you have the same issue trying to upgrade to Laravel 6?

**Comment by PowerKiKi** [comment]
Another alternative to consider would be to transfer this repository under the dompdf organization.

@m-bosch no, I don't use Laravel. I stumble upon this issue because of https://github.com/dompdf/dompdf/pull/1892

**Comment by m-bosch** [comment]
> Another alternative to consider would be to transfer this repository under the dompdf organization.
> 
> @m-bosch no, I don't use Laravel. I stumble upon this issue because of [dompdf/dompdf#1892](https://github.com/dompdf/dompdf/pull/1892)

yes the underlaying package uses dompdf that uses this package... some issue other case :-)

**Comment by PhenX** [comment]
Hello @PowerKiKi, @m-bosch you are right, I moved to C# and don't take time anymore for this project.
I'm ok to add new contributors of course. By the way, what would be the advantage to move the project to the dompdf organization ?

**Comment by m-bosch** [comment]
hi @PhenX, thank you for the response. I see the Dompdf organization isnt that active lately, (no updates this year. i found a PR https://github.com/dompdf/dompdf/pull/1892 about the issue, but they wait for the new stuff i think...

**Comment by m-bosch** [comment]
@PhenX thank you for the merge, will you make a new release so we can update the composers?

**Comment by PowerKiKi** [comment]
The README of this project states that it was created specifically for dompdf. By moving it over there, it would better reflect its origin and purpose. But more importantly it would reduce the bus factor. If you lose interest in the project, then other contributors of the dompdf organisation can easily and naturally keep the project alive.

Because you are member of the dompdf organization, you would lose nothing at all, keep all the credits and existing access, but you would allow other people to possibly manage the project instead of you (including selecting potential new contributors). IMHO oargnisation with a few users are way better for open-source projects.

**Comment by PowerKiKi** [comment]
In this specific case, if this project was under the org, then @bsweeney could have solved the issue himself 6 months ago. Allowing dompdf and php-svg-lib to evolve together seems like a huge win to me.

**Comment by PhenX** [comment]
Sure, I have no problem at all with all this :) Letting the project out of the dompdf org was motivated by the hope that it could be used for other projects (like php-font-lib), I don't know it it's the case.
I invited some of you on the project so that this situation doesn't happen anymore.

**Comment by PowerKiKi** [comment]
Thank you for the invitation ! I accepted it.

However would you consider transferring ownership to the dompdf organization as well ? this would allow the organization to also manage composer/packagist integration, and other services as well (such as Travis and any other things that might appear in the future) ?

That way the project would really be independent of you, but you would still keep all the same access and privileges that you already have.

**Comment by m-bosch** [comment]
composer/packagist For the time being it would be great to make a new version etc so we can fetch it with composer/packagist right? @PowerKiKi

**Comment by PowerKiKi** [comment]
PhenX made it happen: https://packagist.org/packages/phenx/php-svg-lib

Thanks ! :tada:

**Comment by PowerKiKi** [comment]
@PhenX if you still agree to transfer this project under the dompdf organisation, then you should go to https://github.com/PhenX/php-svg-lib/settings, and click on the `Transfer` button:

image

**Comment by PowerKiKi** [comment]
@PhenX it's been a few months since you agree to move this project to the dompdf organization. Could you please do in the next few days ?

This could help situations like the lagging release reported in #59

**Pull request merged**